src/streamsync/serve.py

```diff
@@ -359,8 +362,23 @@ async def stream(websocket: WebSocket):
 
     server_path = os.path.dirname(__file__)
     server_static_path = pathlib.Path(server_path) / "static"
-    asgi_app.mount(
-        "/", StaticFiles(directory=str(server_static_path), html=True), name="server_static")
+    if server_static_path.exists():
+        asgi_app.mount(
+            "/", StaticFiles(directory=str(server_static_path), html=True), name="server_static")
+        asgi_app.state.is_server_static_mounted = True
+    else:
+        logging.error(
+            textwrap.dedent(
```

Comment:
I didn't know `dedent`, that's really nice. Based on this, please improve code formatting wherever you see it -starting with `serve.py` and the "streamsync v0.2.8" message.

Reply:
Got it!